lib/studies/reply/mail_card_preview.dart

```diff
@@ -47,11 +51,87 @@ class MailPreviewCard extends StatelessWidget {
           message: email.message,
           onTap: openContainer,
         );
+
+        if (isDesktop) {
+          return mailPreview;
+        } else {
+          return Dismissible(
+            key: ObjectKey(email),
+            dismissThresholds: const {
+              DismissDirection.startToEnd: 0.4,
+              DismissDirection.endToStart: 1,
+            },
+            onDismissed: (direction) {
+              switch (direction) {
+                case DismissDirection.endToStart:
+                  break;
+                case DismissDirection.startToEnd:
+                  break;
+                default:
+              }
+            },
+            background: _DismissibleContainer(
+              icon: 'twotone_delete',
+              backgroundColor: colorScheme.primary,
+              iconColor: ReplyColors.blue50,
```

Comment:
do these 2 icon colors work in both light and dark mode?

Reply:
Yes, same color for both modes works.